lib/crowbar/client/command/upgrade/database.rb

```diff
@@ -50,7 +50,7 @@ def execute
                 err request.parsed_response["error"]
               end
 
-              response = JSON.parse(request.body)
+              response = JSON.parse(request.body) unless request.body.nil?
```

Comment:
Thanks! Yes, you are right, https://github.com/crowbar/crowbar-client/commit/8358869e124b02c4c1c01f814f3c528d6344da4f should address this.

While looking at https://bugzilla.suse.com/show_bug.cgi?id=1001278 I set up a develcloud6 yesterday and ran an upgrade job to develcloud7 and ran into this problem. The database did not respond and the upgrade just failed with an uncaught exception.

I am surprised I didn't get your commit though. It was committed already 5 days ago. Shouldn't it have been included in the upgrade yesterday?